sys-apps/ndctl/ndctl-64.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 autotools linux-info systemd
+
+DESCRIPTION="Utility library for managing the libnvdimm sub-system in the Linux kernel"
+
+HOMEPAGE="http://pmem.io/"
+SRC_URI="https://github.com/pmem/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RESTRICT="test"
```

Comment:
Why?

Reply:
All the tests not only require root (in fact, pretty much doing anything with the nvdimm subsystem needs root) but special testing kernel modules. Hence all the tests either don't run or are skipped.

Reply:
Can you please put this in a comment so that it's clear from the ebuild?

Reply:
Comment added